playwright/fixtures/next.ts

```diff
@@ -0,0 +1,155 @@
+/* eslint-disable sort-keys */
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+/* eslint-disable no-empty-pattern */
+
+import { AddressInfo } from 'net';
+import { test as base } from '@playwright/test';
+// @ts-ignore
+import fetch from 'node-fetch';
+import moxy from 'moxy';
+import next from 'next';
+import { parse } from 'url';
+import path from 'path';
+import { createServer, Server } from 'http';
+
+import RosaLuxemburg from '../mockData/users/RosaLuxemburg';
+import { Mock, MoxyHTTPMethod, ZetkinAPIResponse } from '../types';
+import { ZetkinSession, ZetkinUser } from '../../src/types/zetkin';
+
+
+interface NextTestFixtures {
+    login: () => Promise<void>;
+}
+
+interface NextWorkerFixtures {
+    next: {
+        appUri: string;
+        moxy: {
+            removeMock: (path?: string, method?: MoxyHTTPMethod, ) => Promise<void>;
+            setMock: <G>(path: string, method: MoxyHTTPMethod, response?: Mock<G>) => Promise<void>;
+        };
```

Comment:
I find it a little unintuitive that `moxy` is a part of the `next` fixture and not it's own fixture. Do you see my point? I would move it into it's own fixture and maybe add a `port` field that can contain the port that was allocated for it.

If I understand things correctly, it would be possible to initialize `moxy` first so that it's made available when the `next` fixture is initialized next?

Reply:
Reworked these to be separate :+1: